utils/bcm27xx-eeprom/Makefile

```diff
@@ -45,15 +69,28 @@ define Package/bcm27xx-eeprom/install
 	$(INSTALL_BIN) $(PKG_BUILD_DIR)/rpi-eeprom-config $(1)/usr/bin
 	$(INSTALL_BIN) $(PKG_BUILD_DIR)/rpi-eeprom-digest $(1)/usr/bin
 	$(INSTALL_BIN) $(PKG_BUILD_DIR)/rpi-eeprom-update $(1)/usr/bin
+endef
 
-	$(INSTALL_DIR) $(1)/lib/firmware/raspberrypi/bootloader
-	$(CP) $(PKG_BUILD_DIR)/firmware/release-notes.md $(1)/lib/firmware/raspberrypi/bootloader
+define Package/bcm2711-eeprom/install
+	$(INSTALL_DIR) $(1)/lib/firmware/raspberrypi/bootloader-2711
+	$(INSTALL_DIR) $(1)/lib/firmware/raspberrypi/bootloader-2711/critical
+	$(INSTALL_DIR) $(1)/lib/firmware/raspberrypi/bootloader-2711/stable
+
+	$(CP) $(PKG_BUILD_DIR)/firmware-2711/release-notes.md $(1)/lib/firmware/raspberrypi/bootloader-2711/
+	$(CP) $(PKG_BUILD_DIR)/firmware-2711/critical/ $(1)/lib/firmware/raspberrypi/bootloader-2711/
+	$(CP) $(PKG_BUILD_DIR)/firmware-2711/stable/ $(1)/lib/firmware/raspberrypi/bootloader-2711/
+endef
 
-	$(INSTALL_DIR) $(1)/lib/firmware/raspberrypi/bootloader/critical
-	$(CP) $(PKG_BUILD_DIR)/firmware/critical/ $(1)/lib/firmware/raspberrypi/bootloader/
+define Package/bcm2712-eeprom/install
+	$(INSTALL_DIR) $(1)/lib/firmware/raspberrypi/bootloader-2712
+	$(INSTALL_DIR) $(1)/lib/firmware/raspberrypi/bootloader-2712/critical
+	$(INSTALL_DIR) $(1)/lib/firmware/raspberrypi/bootloader-2712/stable
 
-	$(INSTALL_DIR) $(1)/lib/firmware/raspberrypi/bootloader/stable
-	$(CP) $(PKG_BUILD_DIR)/firmware/stable/ $(1)/lib/firmware/raspberrypi/bootloader/
+	$(CP) $(PKG_BUILD_DIR)/firmware-2712/release-notes.md $(1)/lib/firmware/raspberrypi/bootloader-2712/
+	$(CP) $(PKG_BUILD_DIR)/firmware-2712/critical/ $(1)/lib/firmware/raspberrypi/bootloader-2712/
+	$(CP) $(PKG_BUILD_DIR)/firmware-2712/stable/ $(1)/lib/firmware/raspberrypi/bootloader-2712/
 endef
 
 $(eval $(call BuildPackage,bcm27xx-eeprom))
+$(eval $(call BuildPackage,bcm2711-eeprom))
+$(eval $(call BuildPackage,bcm2712-eeprom))
```

Comment:
Maybe this should be done in separate commit.

Reply:
This is now a separate commit: b8b0e4549d80c0512084a9a2c82f280950d7ed74?w=1